### PR TITLE
Pass in top level variants to customize build at deploy time

### PR DESCRIPTION
**Pull request by sjmackenzie**

### CODE REVIEW

File: ci/script.sh [hunk 1]
```diff
@@ -6,13 +6,16 @@ set -ex
 main() {
     cross build --target $TARGET --release
 
-    if [ ! -z $DISABLE_TESTS ]; then
-        return
+    if [ -z $DISABLE_TESTS ]; then
+        cross test --target $TARGET --features=integration-test --release
     fi
-
-    cross test --target $TARGET --features=integration-test --release
```
Comment (NicolasDP) [inline ci/script.sh]: why changing this part of the script?

Comment (sjmackenzie) [reply ci/script.sh]:
```
if [ $DISABLE_TESTS is zero length ]; then
   execute tests
fi
```
is the same as 
```
if [ $DISABLE_TESTS is *not* zero length]; then 
    return
fi
Now execute the tests
```
The former is clearer, easier to read and does exactly the same thing as the latter.

Comment (sjmackenzie) [reply ci/script.sh]: Let me add a comment to clarify.

**Comment by sjmackenzie** [comment]
Closing this as it's better I make these changes on my own release branch.

**Pull request closed without merging**